src/etc_as/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+A configurable assembler for ETCa
+"""
+
+__version__ = open(__file__.rstrip('__init__.py') + 'version').read()
```

Comment:
Is there a reason to keep the `version` file? Otherwise I would prefer having the version just inside the `__init__.py` file.

Reply:
It should be fine to embed the version number directly in the python file.